apidef/oas/middleware.go

```diff
@@ -572,6 +572,44 @@ func (tm *TransformRequestMethod) ExtractTo(meta *apidef.MethodTransformMeta) {
 	meta.ToMethod = tm.ToMethod
 }
 
+type TransformRequestBody struct {
+	// Enabled enables transform request body middleware.
+	Enabled bool `bson:"enabled" json:"enabled"`
+	// Format of the request body, xml or json.
+	Format apidef.RequestInputType `bson:"format" json:"format"`
+	// Path file path for the template.
+	Path string `bson:"path" json:"path"`
+	// Body base64 encoded representation of the template.
+	Body string `bson:"body" json:"body"`
+}
+
+func (tr *TransformRequestBody) Fill(meta apidef.TemplateMeta) {
+	if ShouldOmit(meta) {
```

Comment:
This raises up a consistency question. Usually we have the `ShouldOmit` outside of the solitary Fill()'s, but use it to clear a data to nil, for example in `*Server.Fill`. Should this be outside of TransformRequestBody.Fill?

Reply:
you're right.